dist/index.html

```diff
@@ -148,6 +148,10 @@ <h2>Total Cost</h2>
             </form>
             <button type="button" class="addButton" id="add-button">Add to Pantry</button>
         </menu>
+        <div>
+          <span>*Required Field*</span>
```

Comment:
Hey Jenn, have you seen this? It might be better to do instead of the spans? https://developer.mozilla.org/en-US/docs/Web/CSS/:required